include/remill/BC/PCodeCFG.h

```diff
@@ -0,0 +1,91 @@
+/*
+ * Copyright (c) 2022-present Trail of Bits, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+#pragma once
+
+#include <glog/logging.h>
+
+#include <cstddef>
+#include <mutex>
+#include <sleigh/libsleigh.hh>
+#include <unordered_map>
+#include <variant>
+#include <vector>
+
+#include "lib/Arch/Sleigh/ControlFlowStructuring.h"
+
+
+namespace remill {
+namespace sleigh {
+
+
+struct IntrainstructionIndex {
+  size_t target_block_ind;
```

Comment:
`IntrainstructionIndex` is kinda hard to parse at quick glance. Can we come up with something easier? Does it represent instruction index (as in 5th instruction) in the block/function? Or is it more like an address offset? Add doc.

Also, with name as long as `target_block_ind` there is no reason not to include the `ex`. If cutting on letters, I would do something like `trg_block_idx` instead.

Reply:
[71a4ddf](https://github.com/lifting-bits/remill/pull/645/commits/71a4ddf6d96b25b0b09ba908ad4b91fb82624e34)

Renamed the variable but still needs some docs. Could you please add that @2over12?

Reply:
3724c268b07f1f9e69645c44801a423c8a1eaec2

Thanks @2over12.